cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1374,15 +1374,6 @@ def test_advanced_components(self):
         self.assertNotEqual(only_template.get('category'), 'video')
         self.assertNotEqual(only_template.get('category'), 'openassessment')
 
-    def test_advanced_components_without_display_name(self):
```

Comment:
@adampalay I could not find any advanced component that lacks a display name. This test was explicitly created because the GST tool does not have a display name.